src/app/test_executive/genesis_export_test.ml

```diff
@@ -0,0 +1,287 @@
+    and () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:txn_hash2 ~node_included_in:(`Node node) )
+    in
+    let%bind { hash = txn_hash4; _ } =
+      payment ~logger ~node ~sender:fish1 ~receiver:snark_worker1 (mina 25)
+    and { hash = txn_hash5; _ } =
+      payment ~logger ~node ~sender:fish2 ~receiver:snark_worker2 (mina 50)
+    in
+    let%bind () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:txn_hash4 ~node_included_in:(`Node node) )
+    and () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:txn_hash5 ~node_included_in:(`Node node) )
+    in
+    let%bind { hash = txn_hash7; _ } =
+      payment ~logger ~node ~sender:timed_bp ~receiver:fish1 (mina 10)
+    and { hash = txn_hash8; _ } =
+      payment ~logger ~node ~sender:fish2 ~receiver:timed_bp (mina 15)
+    in
+    let%bind () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:txn_hash7 ~node_included_in:(`Node node) )
+    and () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:txn_hash8 ~node_included_in:(`Node node) )
+    in
+    [%log info] "Sending delegation from fish1 (%s) to bp1 (%s)."
+      ( Public_key.Compressed.to_base58_check
+      @@ Public_key.compress fish1.keypair.public_key )
+      ( Public_key.Compressed.to_base58_check
+      @@ Public_key.compress bp1.keypair.public_key ) ;
+    [%log info] "Sending delegation from fish2 (%s) to bp2 (%s)."
+      ( Public_key.Compressed.to_base58_check
+      @@ Public_key.compress fish2.keypair.public_key )
+      ( Public_key.Compressed.to_base58_check
+      @@ Public_key.compress bp2.keypair.public_key ) ;
+    let%bind node_pk = pub_key_of_node node in
+    let%bind { hash = delegation_hash; _ } =
+      Integration_test_lib.Graphql_requests.must_send_delegation
+        ~sender_pub_key:node_pk
+        ~receiver_pub_key:(Public_key.compress timed_bp.keypair.public_key)
+        ~fee:(Amount.to_fee @@ mina 1)
+        ~logger
+        (Network.Node.get_ingress_uri node)
+    in
+    let%bind () =
+      wait_for t
+        (Wait_condition.signed_command_to_be_included_in_frontier
+           ~txn_hash:delegation_hash ~node_included_in:(`Node node) )
+    in
+    let%bind genesis_config =
+      Integration_test_lib.Graphql_requests.export_genesis_ledger ~logger
+        (Network.Node.get_ingress_uri node)
+    in
+    let ledger_accounts =
+      let open Runtime_config in
+      let open Ledger in
+      let ledger = Option.value_exn genesis_config.ledger in
+      match ledger.base with
+      | Accounts accounts ->
+          accounts
+      | _ ->
+          failwith "Expected base to be accounts"
+    in
+    Malleable_error.List.iter ledger_accounts ~f:(fun genesis_account ->
```

Comment:
This test includes a BP key with timed balance. I wonder, what will be slot expected by the end of test and what will be the condition of that timed account?

Reply:
I.e. will it still be vesting or already fully vested.